pkg/infoschema/issyncer/loader.go

```diff
@@ -353,6 +375,7 @@ func (l *Loader) tryLoadSchemaDiffs(useV2 bool, m meta.Reader, usedVersion, newV
 	diffTypes := make([]string, 0, len(diffs))
 	for _, diff := range diffs {
 		if l.skipLoadingDiff(diff) {
+			l.logger.Warn("Skip load a schema diff due to configuration.", zap.Any("diff", diff), zap.Int64("version", diff.Version))
```

Comment:
```suggestion
			l.logger.Warn("skip load a schema diff due to configuration.", zap.Any("diff", diff), zap.Int64("version", diff.Version))
```
seems it's convention when logging to use lower case for the first word unless it's a special word